app/components/avo/fields/files_field/edit_component.html.erb

```diff
@@ -1,5 +1,5 @@
-<%= field_wrapper **field_wrapper_args, full_width: true do %>
-  <%= render Avo::Fields::Common::FilesListViewerComponent.new(field: @field, resource: @resource) if @field.value.present? %>
+<%= field_wrapper **field_wrapper_args, stacked: true do %>
```

Comment:
@adrianthedev  what about make files field stacked attribute true by default but always get the stacked from the field? Someone may want to have this false and would not be possible without monkey patches

Reply:
Same apply on show component

Reply:
Yes. My thoughts exactly. Have the stackable option control it for the wrapper too.